Tools/CISupport/ews-build/events.py

```diff
@@ -147,11 +197,19 @@ def buildFinished(self, key, build):
         builder = yield self.master.db.builders.getBuilder(build.get('builderid'))
         builder_display_name = builder.get('description')
 
+        patch_id = self.extractProperty(build, 'patch_id')
```

Comment:
Can move this patch_id statement below github check (just before you are validating patch_id).